### PR TITLE
Created basic_queue.py

### PR DESCRIPTION
**Pull request by SawshaDev**
Adds an example on how to use basic queues.

### CODE REVIEW

File: examples/basic_queue.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            await vc.queue.put_wait(search)
+            await vc.play(search)
+            await ctx.send(f'Added `{search.title}` to the queue...', delete_after=10)
+            
+        else:
+            await vc.queue.put_wait(search)
+            await ctx.send(f'Added `{search.title}` to the queue...', delete_after=10)
+
+        
+
```
Comment (EvieePy) [inline examples/basic_queue.py:75]: Why is there so much whitespace here?

File: examples/basic_queue.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    
+    @commands.command()
+    async def queue(self, ctx: commands.Context):
+      vc: wavelink.Player = ctx.voice_client
+
+      if not vc:
+          return await ctx.send('No queue as we are not connected', delete_after=5)
+
+      await ctx.send(vc.queue)
```
Comment (EvieePy) [inline examples/basic_queue.py:85]: Maybe make a prettier, more reliable output here. Queue could be filled with thousands of songs for instance.

File: examples/basic_queue.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        bot.loop.create_task(self.connect_nodes())
+
+    async def connect_nodes(self):
+        """Connect to our Lavalink nodes."""
+        await self.bot.wait_until_ready()
+
+        await wavelink.NodePool.create_node(bot=self.bot,
+                                            host='0.0.0.0',
+                                            port=2333,
+                                            password='YOUR_LAVALINK_PASSWORD')
```
Comment (EvieePy) [inline examples/basic_queue.py:46]: Whitespace?

File: examples/basic_queue.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+    @commands.Cog.listener()
+    async def on_wavelink_node_ready(self, node: wavelink.Node):
+        """Event fired when a node has finished connecting."""
+        print(f'Node: <{node.identifier}> is ready!')
+
+
```
Comment (EvieePy) [inline examples/basic_queue.py:53]: Whitespace?

**Review by EvieePy — CHANGES_REQUESTED** [review]

File: examples/basic_queue.py (hunk 3 above)
Comment (EvieePy) [reply examples/basic_queue.py:46]: Should be one newline

File: examples/basic_queue.py (hunk 4 above)
Comment (EvieePy) [reply examples/basic_queue.py:53]: Should be one newline

Comment (SawshaDev) [reply examples/basic_queue.py:53]: ah shit, sorry for all that, all this mistakes were due to how i used the shit in my IDE. will fix all these soon.

**Comment by EvieePy** [comment]
Closing this as this is for an outdated version of Wavelink. Feel free to open another PR for the latest version. Thanks though 👍

**Pull request closed without merging**